spec/node_spec.rb

```diff
@@ -43,21 +38,86 @@ def node(name)
   let :testnode02 { node('testnode02') }
   let :testnode03 { node('testnode03') }
 
-  describe '#configs' do
-    it 'returns possible configs for node in precedence order' do
-      expect(testnode01.configs).to eq(["domain", "cluster", "nodes", "testnodes", "testnode01"])
+  before do
+    SpecUtils.use_mock_genders(self)
+  end
+
+  # XXX adapt these to use FakeFS and make dependencies explicit?
+  context 'without using FakeFS' do
```

Comment:
Yep, in the final PR the `Node#answers` test do. I've been including these comments when I spot some tests which depend on the file system but do not use `FakeFS`, and instead use some other means of working with paths, to remind us to fix this if these are touched again.